src/main/scala/EmrSparkPlugin.scala

```diff
@@ -57,6 +63,8 @@ object EmrSparkPlugin extends AutoPlugin {
     val sparkTerminateCluster = taskKey[Unit]("terminate cluster")
     val sparkSubmitJob = inputKey[Unit]("submit the job")
     val sparkSubmitJobWithMain = inputKey[Unit]("submit the job with specified main class")
+    val sparkSubmitSteps = taskKey[Unit]("submit multiple steps")
+    val sparkUploadJarToS3 = taskKey[Unit]("Upload jar to S3")
```

Comment:
I was looking to separate createCluster, uploadArtifacts and startSteps. Still not complete. Yeah forgetting to upload jars is a problem. Don't you think that will show up during the first run of the release process or sbt commands? It will be a instant failure that can be addressed.